### ISSUE TITLE
Provide a Terracotta configuration supporting active-passive mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/terrastore#3]
```
While adding active-passive to the current default configuration should be
fairly easy, providing a ready-to-use configuration file may help
Terracotta newbies.
```

Original issue reported on code.google.com by `sergio.b...@gmail.com` on 28 Nov 2009 at 12:10
- Blocked on: #20

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 4 Dec 2009 at 9:39
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Feature**

**Comment by GoogleCodeExporter** [comment]
```
Explained in: http://code.google.com/p/terrastore/wiki/Cluster_Operations
```

Original comment by `sergio.b...@gmail.com` on 22 Dec 2009 at 4:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 22 Dec 2009 at 4:57
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in terrastore-install tool.
```

Original comment by `sergio.b...@gmail.com` on 28 Dec 2009 at 11:48
- Changed state: **Accepted**
- Added labels: **Milestone-0.3**

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 28 Dec 2009 at 11:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 28 Dec 2009 at 11:53
- Added labels: **Priority-Medium**
- Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 28 Dec 2009 at 1:44
- Changed state: **Fixed**
